pymc/tests/test_distributions_random.py

```diff
@@ -1502,6 +1502,40 @@ def seeded_zero_inflated_negbinomial_rng_fn(self):
     ]
 
 
+class TestZeroInflatedNegativeBinomial(BaseTestDistribution):
```

Comment:
This test can be much smaller. Check other tests for alternative parametrizations in this module. Like `NormalTau`

Reply:
After a quick glance on `NormalTau`, we only need to test on `check_pymc_params_match_rv_op`?

Reply:
Yes